__tests__/app.test.js

```diff
@@ -86,3 +86,29 @@ describe("GET", () => {
       });
   });
 });
+
+describe("GET", () => {
+  it("should respond with an array of review objects with expected properties", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        const bodyReviews = body.reviews;
+        expect(bodyReviews).toHaveLength(13);
+        expect(bodyReviews).toBeInstanceOf(Array);
```

Comment:
Using .forEach in the next line means that reviews has to be an array for this test to work, so line 98 isn't needed.